words/source/GamePane.js

```diff
@@ -387,14 +371,15 @@ defineParticle(({ DomParticle, resolver }) => {
         }
       }
       state.move.coordinates = newCoordinates;
+      // TODO(wkorman): Consider making Move purely state.
+      this._setMove({ coordinates: newCoordinates });
```

Comment:
This one line is the primary fix. The rest was leaderboard work and minor cleanup which I was doing already, and I ended up just putting it all in this one change for expediency.